packages/x-date-pickers/src/locales/esES.ts

```diff
@@ -3,32 +3,32 @@ import { getPickersLocalization } from './utils/getPickersLocalization';
 import { TimeViewWithMeridiem } from '../internals/models';
 
 const views: Record<TimeViewWithMeridiem, string> = {
-  hours: 'las horas',
-  minutes: 'los minutos',
-  seconds: 'los segundos',
-  meridiem: 'meridiano',
+  hours: 'Horas',
```

Comment:
Nitpick: Technically having this word start in lowercase seems a bit more appropriate, given that it is used mid-sentence.
But that's not a problem now, because those cases are only used on `aria-label` attributes that will not be visible, just read by screen readers. 👌